static/src/stylesheets/module/football/_overview.scss

```diff
@@ -1,3 +1,160 @@
+/* ==========================================================================
+   World Cup 2018
+   ========================================================================== */
+.wc-overview {
+    .container__title {
+        &:after {
+            display: none;
+        }
+    }
+}
+
+.facia-header {
+    .facia-header__inner {
```

Comment:
Does everything here need to be nested inside `.facia-header` for specificity reasons?